controllers/persistentvolume/persistentvolume_controller.go

```diff
@@ -208,43 +211,72 @@ func (r *PersistentVolumeWatcher) ensureStorageSystemLabel(pv *corev1.Persistent
 func (r *PersistentVolumeWatcher) getPVManagementAddress(pv *corev1.PersistentVolume) (string, error) {
 	r.Log.Info("looking for PersistentVolume management address")
 
-	pvVolumeAttributes := pv.Spec.CSI.VolumeAttributes
-	pvMgmtAddr, ok := pvVolumeAttributes[util.PVMgmtAddrKey]
+	var pvMgmtAddr string
+	pvVolumeHandle := pv.Spec.CSI.VolumeHandle
```

Comment:
consider changing the location of the line inside the topology if statement( before checking if it's empty)

Reply:
done